src/shared/Divider/Divider.tsx

```diff
@@ -0,0 +1,34 @@
+import styled from '@emotion/native';
+import React from 'react';
+import { ThemeProps } from 'theme';
+
+interface DividerProps {
+  width?: number;
+  backgroundColor?: string;
+  marginTop?: number;
+  marginBottom?: number;
+}
+
+export default function Divider({
+  width,
+  backgroundColor,
+  marginTop,
+  marginBottom,
+}: DividerProps) {
+  return (
+    <AdaptiveDivider
+      width={width}
+      backgroundColor={backgroundColor}
+      marginTop={marginTop}
+      marginBottom={marginBottom}
+    />
+  );
+}
+
+const AdaptiveDivider = styled.View<DividerProps & ThemeProps>`
+  background-color: ${(props) => props.backgroundColor ?? props.theme.text};
+  margin: ${(props) => props.marginTop ?? 0}px auto
+    ${(props) => props.marginTop ?? 0}px auto;
```

Comment:
```suggestion
  margin: ${(props) => props.marginTop ?? 0}px auto
    ${(props) => props.marginTop ?? 0}px;
```

맨 뒤에 auto는 빼도 될듯

Reply:
```css
margin: 15px auto 15px auto;
```
이거랑

```css
margin: 15px auto 15px;
```
랑

같나?

Reply:
ㅇㅇ 같음

Reply:
글고 지금 보니까 
```
margin: 15px auto;
```

이렇게 쓰면 될듯

Reply:
아 내가 코드를 잘못 씀

```suggestion
  margin: ${(props) => props.marginTop ?? 0}px auto
    ${(props) => props.marginBottom ?? 0}px auto;
```
이렇게 해야됨

Reply:
verticalMargin으로 변경 할 것이므로 

```css
margin: 15px auto;
```

로 변경 예정